dags/common/cleanup.py

```diff
@@ -18,7 +20,37 @@ def convert_html_subsripts_to_latex(input):
 def clean_collaboration(input):
     return clean_whitespace_characters(input.replace("for the", ""))
 
+
 def remove_specific_tags(value, tags=None, attributes=None):
     tags = tags or []
     attributes = attributes or []
     return clean(value, tags=tags, attributes=attributes, strip=True)
+
+
+class RemoveUnneccessaryTagsFilter(Filter):
+    """Filter removes content of label, but not the tag itself!"""
+
+    def __iter__(self):
+        label_tag = False
+        for token in Filter.__iter__(self):
+            if token["type"] == "StartTag" and token["name"] == "label":
+                label_tag = True
+            elif token["type"] == "EndTag":
+                label_tag = False
+            elif label_tag:
+                token["data"] = ""
```

Comment:
shouldn't be something like:

```suggestion
            if label_tag:
                token["data"] = ""
                label_tag = False
```